resolvconf/resolvconf.go

```diff
@@ -52,7 +53,11 @@ type File struct {
 func Get() (*File, error) {
 	resolv, err := ioutil.ReadFile("/etc/resolv.conf")
 	if err != nil {
-		return nil, err
+		if os.IsNotExist(err) {
+			logrus.Infof("/etc/resolv.conf does not exist. Using default external servers : %v", defaultIPv4Dns)
```

Comment:
I guess something should be returned here? Otherwise this function still attempts to read the file below